### PR TITLE
Development

### PR DESCRIPTION
**Pull request by AlejoCode**
Hi.

Thank you so much for your help

@zenott and I ended the project and wanted to know feedback about you guys

Cheers

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,234 @@
+body {
+  margin: 0px;
+  padding: 0px;
+}
+
+.container {
+  width: 1440px;
```
Comment (zlayabekrija) [inline assets/css/style.css]: I guess that you wanted to have max-width. With this width it looks way to big for my screen, please fix it. My screen size is 1280px.

File: index.html [hunk 2]
```diff
@@ -12,5 +12,75 @@
...
+
+          <footer id="homefooter" data-hires="true">
+
+              <div class="wrapper">
+                  <p class="left">Shop the <a href="https://web.archive.org/web/20140301004610/http://store.apple.com/us" >Apple Online Store</a> (1-800-MY-APPLE), visit an <a href="/web/20140301004610/http://www.apple.com/retail/" >Apple Retail Store</a>, or find a <a href="/web/20140301004610/http://www.apple.com/buy/" >reseller</a>.</p>
+                  <ul class="piped right"><li class="first">
+                    <a href="/web/20140301004610/http://www.apple.com/sitemap/">Site Map</a></li><li>
+                    <a href="/web/20140301004610/http://www.apple.com/hotnews/" >Hot News</a></li><li>
+                    <a href="/web/20140301004610/http://www.apple.com/rss/" >RSS Feeds</a></li><li>
+                    <a href="/web/20140301004610/http://www.apple.com/pr/" ></a>Media Info</a></li><li>
```
Comment (zlayabekrija) [inline index.html]: one extra `</a>` tag

File: index.html [hunk 3]
```diff
@@ -12,5 +12,75 @@
       href="assets/css/style.css"
     />
   </head>
-  <body></body>
+  <body>
+    <div class="container">
+      <section class="hero">
```
Comment (zlayabekrija) [inline index.html]: By convention `<section>` tag requires heading tag (h1,h2...). Please use header,main,aside tags ...
I would call this header. Convetion isn't must it's only good practise.

File: index.html [hunk 4]
```diff
@@ -12,5 +12,75 @@
...
+          <img src="assets/images/promo_30_years.jpg" alt="promo_30_years" />
+        </div>
+        <div class="down-elem">
+          <img
+            src="assets/images/promo_supplier_responsibility.jpg"
+            alt="promo_supplier_responsibility"
+          />
+        </div>
+      </section>
+      <section class="footer">
```
Comment (zlayabekrija) [inline index.html]: please use footer instead of section

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Great work guys, some small things to change, please follow them up.
Please make read me file, it's very important for the project.
Please follow the notes in the code.
Please fix the required notes, and re submit review request.
Happy coding

**Comment by AlejoCode** [comment]
Hi we made the changes and need approval please

**Review by zlayabekrija — APPROVED** [review]
Great work, thumbs up 👍  from me.
Happy coding :)

**Pull request merged**